pkg/controllers/clusterregistrationtoken/handler.go

```diff
@@ -195,6 +195,6 @@ func (h *handler) deleteExpired(token *fleet.ClusterRegistrationToken) (bool, er
 		return true, h.clusterRegistrationTokens.Delete(token.Namespace, token.Name, nil)
 	}
 
-	h.clusterRegistrationTokens.EnqueueAfter(token.Namespace, token.Name, expire.Sub(time.Now()))
+	h.clusterRegistrationTokens.EnqueueAfter(token.Namespace, token.Name, time.Until(time.Now()))
```

Comment:
Same comment as above - what is the linter error for using Sub() method on time package?

Reply:
```
pkg/controllers/clusterregistrationtoken/handler.go:198:72: S1024: should use time.Until instead of t.Sub(time.Now()) (gosimple)
	h.clusterRegistrationTokens.EnqueueAfter(token.Namespace, token.Name, expire.Sub(time.Now()))
```

https://staticcheck.io/docs/checks/#S1024